pom.xml

```diff
@@ -0,0 +1,208 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.exist-db</groupId>
+        <artifactId>exist-apps-parent</artifactId>
+        <version>1.3</version>
+        <relativePath/>
+    </parent>
+
+    <groupId>org.exist-db</groupId>
+    <artifactId>atomic-wiki</artifactId>
+    <version>0.7-SNAPSHOT</version>
```

Comment:
current public-repo app seems to be at 0.8, so we should one up this to `0.9-snapshot`

Reply:
Well this was previously `0.6` in GitHub which is why I set it to `0.7-SNAPSHOT`. I will change it...

Reply:
yeah not sure why public-repo offers 0.8 when it says 0.6 in here, but it does